pkgs/models/model.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenvNoCC
+, fetchurl
+, makeWrapper
+
+, sticker
+
+# Word and tag embeddings as an attrset, containing the attributes:
+#
+# - embeds: Embedding file fixed-output derivation.
+# - filename: Embedding file name in the model's sticker.conf
+, wordEmbeds
+, tagEmbeds ? null
```

Comment:
`? null` makes tags optional, in the German UD POS model those are left out for obvious reasons

Reply:
Yes, in an attribute set that is a function argument (such as here), `?` is used to specify a default value for attributes that are absent. You could also have e.g. `foo ? "bar"` and `foo` would be the string `"bar"` if `foo` is absent in the attribute set.